src/company_base.h

```diff
@@ -92,7 +92,7 @@ struct CompanyProperties {
 	 */
 	bool is_ai;
 
-	Money yearly_expenses[3][EXPENSES_END];                ///< Expenses of the company for the last three years, in every #ExpensesType category.
+	std::array<Expenses, 3> yearly_expenses{}; ///< Expenses of the company for the last three years, in every #ExpensesType category.
```

Comment:
nitpick: ", in every .." feels redundant here, now you use the type `Expenses`?

Reply:
Oh yes, you are right. I effectively moved the "in every ..." to the alias definition so it is redundant here.